docs/src/test/java/docs/http/javadsl/HttpClientExampleDocTest.java

```diff
@@ -39,6 +39,22 @@
 import akka.http.javadsl.model.headers.SetCookie;
 //#collecting-headers-example
 
+
+//#single-request-in-actor-example
+import akka.actor.AbstractActor;
```

Comment:
Really cool to add the imports as well for the Java version!

Reply:
Great 👍